docs/language-highlight.md

```diff
@@ -8,4 +8,25 @@
 <script src="//cdn.jsdelivr.net/npm/prismjs/components/prism-php.min.js"></script>
 ```
 
+To use the new languages, make sure the code block label matches the part after `prism-` in the file name. FOr example, for `prism-bash.js` write a code block labeled with `bash` like this:
+
+````
+```bash
+echo "hello"
+```
+````
+
+?> Note that with GitHub-flavored markdown, `sh` and `bash` are effectively aliases of each other, but this is not the case with Prism. So using `sh` will not enable `bash` syntax in this case.
```

Comment:
This is confusing for a few reasons:

1. GitHub-flavored markdown doesn't handle syntax highlighting (see [spec](https://github.github.com/gfm/)). I think you're referring to how GitHub handles markdown.
1. Docsify supports GitHub-flavored markdown (see the [Supported Markdown specifications](https://marked.js.org/#/README.md) for [marked](https://github.com/markedjs/marked))
1. [Prism's source](https://github.com/PrismJS/prism/blob/c6c62a69a06e80535c387f4041da80e23a3654f0/plugins/file-highlight/prism-file-highlight.js#L18) lists `sh` as an alias for `bash`.
1. [Prism's docs](https://prismjs.com/#supported-languages) list `shell` as an alias for `bash` (and `sh` is an alias for `shell`).

I suggest we just delete the `?> Note...` section.

Reply:
> GitHub-flavored markdown doesn't handle syntax highlighting

Maybe I should say "Markdown on GitHub".

Reply:
> Prism's source lists sh as an alias for bash.

When I add `prism-bash.js` and use `sh` as the syntax it doesn't highlight. Same with `shell`. `bash` works though. Is it new? Does it work for you? Maybe I need an update.

Reply:
I was incorrect about `sh` being an alias for `shell` and `bash` in the Prism source (it was late 🤷 ). Prism simply associates `sh` as a file extension for bash scripts.

The difference between GitHub and Prism highlighting is:

* Prism requires using the alias(es) listed in their [docs](https://prismjs.com/#supported-languages). For Bash, Prism lists only `bash` and `shell`. I can confirm both `bash` and `shell` work as expected in docsify-themeable, while `sh` does not work:

  <img width="445" alt="Screen Shot 2020-06-28 at 11 24 09 PM" src="https://user-images.githubusercontent.com/442527/85973609-e9637b80-b998-11ea-866a-718f2c21e8d6.png">

* GitHub (it appears) uses file extensions (like `sh`) in addition to the usual aliases. Notice how the same markdown from the screenshot above renders here on GitHub (specifically, the proper `sh` highlighting):

  ```bash
  # bash
  echo "hello"
  ```

  ```shell
  # shell
  echo "hello"
  ```

  ```sh
  # sh
  echo "hello"
  ```

   To confirm this, I tried to find another file type that Prism associates an extension with but does not list as an alias. Batch has a file extension of `bat` in the [source](https://github.com/PrismJS/prism/blob/c6c62a69a06e80535c387f4041da80e23a3654f0/plugins/file-highlight/prism-file-highlight.js#L19) but only lists `batch` as an alias in the [docs](https://prismjs.com/#supported-languages) so this is a good match. Here's how `batch` and `bat` render on docsify-themeable:

  <img width="436" alt="Screen Shot 2020-06-29 at 12 20 17 AM" src="https://user-images.githubusercontent.com/442527/85975731-7230e600-b99e-11ea-9ad7-2c7d876e34a4.png">

  And here's how that above markdown renders here on GitHub:

  ```batch
  :: batch
  @ECHO "HELLO"
  ```

  ```bat
  :: bat
  @ECHO "HELLO"
  ```

Summary: Docsify uses Prism aliases only, GitHub uses language aliases and extensions.